### PR TITLE
Fixed some guns erroneously fitting inside handgun holsters (notably, ion rifles)

### PR DESCRIPTION
**Pull request by 9600bauds**
:cl:
 * bugfix: Fixed some guns erroneously fitting inside handgun holsters (notably, ion rifles)

### CODE REVIEW

File: code/modules/projectiles/guns/projectile/nagant.dm [hunk 1]
```diff
@@ -87,6 +87,9 @@
 	w_class = W_CLASS_MEDIUM
 	slot_flags = SLOT_BELT
 
+/obj/item/weapon/gun/projectile/nagant/obrez/isHandgun()
+	return TRUE //WHY NOT
```
Comment (Intigracy) [inline code/modules/projectiles/guns/projectile/nagant.dm:91]: :ok_hand:

File: code/modules/projectiles/guns/projectile/sniper.dm [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /obj/item/weapon/gun/projectile/hecate
-	name = "\improper PGM H�cate II"
-	desc = "An Anti-Materiel Rifle. You can read \"Fabriqu� en Haute-Savoie\" on the receiver. Whatever that means..."
+	name = "\improper PGM H�cate II"
```
Comment (Intigracy) [inline code/modules/projectiles/guns/projectile/sniper.dm]: ?

**Comment by Intigracy** [comment]
gun/energy and gun/energy/laser are both used in the maps, they're the energy gun and laser guns found in the armory and on certain heads (HoS, HoP, Cap).

Is leaving them not handgun size intentional?

**Comment by Sprok0** [comment]
the energy gun is actually gun/energy/gun, gun/energy is some sort of taser that uses the old energy gun sprite

File: code/modules/projectiles/guns/projectile/sniper.dm (hunk 2 above)
Comment (9600bauds) [reply code/modules/projectiles/guns/projectile/sniper.dm]: Nuke the french

**Comment by Intigracy** [comment]
My mistake then, but isn't /laser still the laser gun in the armory?

**Comment by Sprok0** [comment]
Yeah, the base laser gun is the one that's usually used.

**Comment by 9600bauds** [comment]
The laser guns are more akin to laser rifles than laser handguns

**Comment by Probe1** [comment]
edd5e9d2-c5f4-11e6-9fa1-52f1151ca4ce

File: code/modules/projectiles/guns/energy/nuclear.dm [hunk 3]
```diff
@@ -16,7 +16,7 @@
 /obj/item/weapon/gun/energy/gun/isHandgun()
 	return TRUE
 
-/obj/item/weapon/gun/energy/gun/attack_self(mob/living/user as mob)
+/obj/item/weapon/gun/energy/gunattack_self(mob/living/user as mob)
```
Comment (Shadowmech88) [inline code/modules/projectiles/guns/energy/nuclear.dm]: This looks like a typo.

**Comment by 9600bauds** [comment]
i'm sure travis would have caught it 8)

**Comment by Probe1** [comment]
So I have to ask before I hit approve.

Why the hell didn't you just set the holsters max w_class to small?

most handguns are ``` w_class = W_CLASS_SMALL ``` already and changing that wouldn't be too big of a deal.

**Comment by 9600bauds** [comment]
For stuff like the Obrez I guess

**Comment by Probe1** [comment]
You could have just made it small.  😢

**Comment by Sprok0** [comment]
The majority of handguns are normal sized, it's just the RnD laser/plasma pistols and the stechtkin that are small

**Comment by 9600bauds** [comment]
Yeah that too, eguns don't fit in boxes do they

**Comment by Probe1** [comment]
Then why would they fit in holsters?  They don't fit in belts either.

**Comment by 9600bauds** [comment]
'cuz they're handgun-shaped of course!

**Pull request merged**